examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -352,81 +400,102 @@ def _mean_frequency_by_risk_group(y_true, y_pred, sample_weight=None,
+    axi.plot(q, y_true_seg, marker='o', linestyle="--", label="observations")
     axi.set_xlim(0, 1.0)
-    axi.set_ylim(0, 0.6)
+    axi.set_ylim(0, 0.5)
     axi.set(
-        title=model[-1].__class__.__name__,
+        title=model[-1],
         xlabel='Fraction of samples sorted by y_pred',
         ylabel='Mean Frequency (y_pred)'
     )
     axi.legend()
 plt.tight_layout()
 
-##############################################################################
-# The ``Ridge`` regression model can predict very low expected frequencies
-# that do not match the data. It can therefore severly under-estimate the risk
-# for some policyholders.
+###############################################################################
+# The dummy regression model predicts a constant frequency. This model does not
+# attribute the same tied rank to all samples but is none-the-less globally
+# well calibrated (to estimate the mean frequency of the entire population).
 #
-# ``PoissonRegressor`` and ``RandomForestRegressor`` show better consistency
-# between predicted and observed targets, especially for low predicted target
-# values.
+# The ``Ridge`` regression model can predict very low expected frequencies that
+# do not match the data. It can therefore severly under-estimate the risk for
+# some policyholders.
 #
-# However, for some business applications, we are not necessarily interested
-# in the ability of the model to predict the expected frequency value, but
-# instead to predict which policyholder groups are the riskiest and which are
-# the safest. In this case, the model evaluation would cast the problem as a
-# ranking problem rather than a regression problem.
+# ``PoissonRegressor`` and ``HistGradientBoostingRegressor`` show better
+# consistency between predicted and observed targets, especially for low
+# predicted target values.
 #
-# To compare the 3 models within this perspective, one can plot the fraction of
+# The sum of all predictions also confirms the calibration issue of the
+# ``Ridge`` model: it under-estimates by more than 3% the total number of
+# claims in the test set while the other three models can approximately recover
+# the total number of claims of the test portfolio.
+#
+# Evaluation of the ranking power
+# -------------------------------
+#
+# For some business applications, we are interested in the ability of the model
+# to rank the riskiest from the safest policyholders, irrespective of the
+# absolute value of the prediction. In this case, the model evaluation would
+# cast the problem as a ranking problem rather than a regression problem.
+#
+# To compare the 3 models from this perspective, one can plot the fraction of
 # the number of claims vs the fraction of exposure for test samples ordered by
-# the model predictions, from safest to riskiest  according to each model:
+# the model predictions, from safest to riskiest according to each model.
```

Comment:
Saying that the exposure and actual claims are cumulative seems important. Maybe:

> To compare the 3 models from this perspective, one can plot the cumulative proportion of claims vs the cumulative proportion of exposure for the test samples order by the model predictions, from safest to riskiest according to each model.

And then an update on the axis as well.

Reply:
Done in 8f80a4d .